tests/test_omegaconf.py

```diff
@@ -91,7 +91,7 @@ def test_is_missing(
 ) -> None:
     cfg = OmegaConf.create(cfg)
     with expectation:
-        cfg.get(key)
+        cfg[key]
```

Comment:
In commit https://github.com/omry/omegaconf/pull/484/commits/0cec9053ab3d630cea6816fbd1b7d6d8278e2416 I made a change to `DictConfig.get`, so that its behavior is consistent with standard `dict.get`:
by default, `cfg.get(key)` should return `None` if `key` is not found.

This change required the above modification to the `test_omegaconf.py` file, using `__getitem__` for testing instead of `get`.

Reply:
Gotcha.
This makes sense and is exactly why I wanted to have more disciplined testing for consistency with plain dict.

Under what circumstances the behavior is changing?
should we include a news fragment for it?

Reply:
The behavior is changed when `DictConfig.get` is called without a default value and `key` is not found. Previously a `ConfigKeyError` was raised; now `None` is returned.

I'll add a news fragment.

Reply:
I also created an Issue documenting the inconsistency: #527